src/bit_machine/mod.rs

```diff
@@ -510,20 +523,28 @@ impl BitMachine {
     }
 }
 
-/// A type that keeps track of which case branches were executed
-/// during the execution of the Bit Machine.
+/// A type that keeps track of Bit Machine execution.
 ///
-/// The trait is implemented for [`SetTracker`], which does the actual tracking,
+/// The trait is implemented for [`SetTracker`], that tracks which case branches were executed,
 /// and it is implemented for [`NoTracker`], which is a dummy tracker that is
```

Comment:
In 4821b6537c3578714924efb84b36ac83694aa365

In my local CI I run

```bash
export 'RUSTDOCFLAGS=--cfg docsrs -D warnings -D rustdoc::broken-intra-doc-links'
cargo doc -j1 --all-features
```

which errors out about the references to `SetTracker` and `NoTracker` which are private types, since these are public docs. We should do this in the real CI as well.

Reply:
Added CI job